dce/main.go

```diff
@@ -330,10 +344,25 @@ func (exec *dockerComposeExecutor) Error(driver exec.ExecutorDriver, err string)
 	log.Printf("Got error message : %s", err)
 }
 
+func validateComposeFiles() error {
+	logger.Println("====================Validating Compose Files====================")
+
+	if !config.SkipPullImages() {
```

Comment:
Why compose file validation is skipped when SkipPullImages is false?

Reply:
Will change that. It shouldn't be skipped.